pkg/k8s/annotations/annotations.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 D2iQ, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package annotations
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// Get will get the value of the supplied annotation.
+func Get(obj metav1.Object, name string) (value string, found bool) {
+	annotations := obj.GetAnnotations()
+	if len(annotations) == 0 {
+		return "", false
+	}
+
+	value, found = annotations[name]
+
+	return
+}
```

Comment:
nit: only use named returns if really necessary, and prefer explicit return values

Also don't need length check, even if annotations is nil.

```suggestion
func Get(obj metav1.Object, name string) (string, bool) {
	annotations := obj.GetAnnotations()
	val, found := annotations[name]
	return val, found
}
```